presto-main/src/main/java/com/facebook/presto/sql/gen/InCodeGenerator.java

```diff
@@ -75,12 +98,14 @@ public ByteCodeNode generateExpression(Signature signature, ByteCodeGeneratorCon
                 Object object = constant.getValue();
                 constantValuesBuilder.add(object);
```

Comment:
@haozhun , i'm not sure if i understand this comment correctly, but constant values are required to build the switch block if areSmallPrimitives is set